src/App.tsx

```diff
@@ -37,6 +40,12 @@ class App extends Component {
       await initializeAccount();
     }
 
+    if (config?.integrations?.inplayer?.clientId) {
+      // this is temporary until the config is loaded from jw side
+      InPlayer.setConfig(import.meta.env.APP_INPLAYER_SDK);
+      await initializeInPlayerAccount();
```

Comment:
You might want to sync this with the latest PR, which cleans up and refactors the app init code: https://github.com/jwplayer/ott-web-app/pull/197

Reply:
I guess I'll wait for it to be merged with develop